.dev/docker/default.conf

```diff
@@ -0,0 +1,51 @@
+# If using Ubuntu this file should be placed in:
+# 			/etc/nginx/sites-available/
+#
+# If using CentOS this file should be placed in:
+#				 /etc/nginx/conf.d/
+#
+server {
+    listen 80;
+    server_name _;
+    
+    root /app/public;
```

Comment:
Security wise it is best practice to not reveal what web server you run. It is only an option if you change the code and compile your own version of nginx .

This setting is the second best it removes the version number from the server header.
```
server_tokens off; # Hide nginx's version
```